src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AtclauseOrderCheck.java

```diff
@@ -208,9 +208,7 @@ public void setTarget(String... targets) {
      */
     public void setTagOrder(String... orders) {
         final List<String> customOrder = new ArrayList<>(orders.length);
-        for (String order : orders) {
-            customOrder.add(order.trim());
-        }
+        customOrder.addAll(Arrays.asList(orders));
         tagOrder = customOrder;
```

Comment:
can it be simply `tagOrder = Arrays.asList(orders);` ?

Reply:
Done